work1-1.html

```diff
@@ -1 +1,16 @@
-text1
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+  <body>
+    <script>
+	for(let i = 3; i <= 99; i ++){
+　　if(i%15 === 0){
```

Comment:
以下のようにスペースを空けるのが慣習ですので、こちらも修正してみましょう！
```suggestion
　　if (i %15 === 0) {
```
他の箇所も同様です！